Atomic/help.py

```diff
@@ -0,0 +1,41 @@
+from . import Atomic
+import subprocess
+from pydoc import pager
+from os import environ
+from . import mount
+import sys
+
+class AtomicHelp(Atomic):
+    def help(self):
+        """
+        Displays the image/containe help file in a pager
+        :return: None
+        """
+        # Set the pager to less -R
+        dm = mount.DockerMount("/tmp")
+        enc=sys.getdefaultencoding()
+        environ['PAGER'] = '/usr/bin/less -R'
+        docker_id = self.get_input_id(self.args.docker_object)
+        self.inspect = self._inspect_image(docker_id)
```

Comment:
When the docker_id is a container, I am using DM._clone() which generates a temporary image from the container given and then run a temporary container.  The temporary container and image is cleaned up after execution.  This very much follows other mount operations.  In the case of using a container, I use help_id to differentiate between the docker_id and the ID of the temporary container.